src/hooks/useAppConfig.test.tsx

```diff
@@ -42,17 +48,21 @@ beforeEach(() => {
 
 test('configured appTiles are returned', async () => {
   const mockAppTiles = ['appTile-1', 'appTile-2', 'appTile-3'].map(mockAppTile);
-  api.mock('GET /v1/life-research/projects/:projectId/app-config', {
-    status: 200,
-    data: {
-      homeTab: {
-        appTiles: mockAppTiles,
-      },
+
+  // Bug in mocker seems to not be able to tell /:id and /list apart
```

Comment:
I think it is thinking that `list` is the `:id`. Did you try swapping the definition order of the route mocks? It seems like `msw/node` [prepends](https://mswjs.io/docs/api/setup-worker/use/) the routes so I would think/hope that mocking `/:id` then mocking `/list` would produce the desired result. Regardless though this seems fine.

Reply:
I'd expect that if `/list` matches multiple routes that the mocking library would prefer the exact match over the wildcard which is sort of why I described this as a bug but perhaps you are right about that.